js/apps/system/_admin/aardvark/APP/react/src/views/views/searchAliasView/EditableViewNameField.tsx

```diff
@@ -0,0 +1,60 @@
+import { CheckIcon, EditIcon } from "@chakra-ui/icons";
+import {
+  IconButton,
+  Input,
+  Stack,
+  Text,
+  useDisclosure
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import { ViewPropertiesType } from "./useFetchViewProperties";
+
+
+export const EditableViewNameField = ({
+  view, isAdminUser, isCluster, setCurrentName
+}: {
+  view: ViewPropertiesType;
+  isAdminUser: boolean;
+  isCluster: boolean;
+  setCurrentName: (name: string) => void;
+}) => {
```

Comment:
Please extract the PropType.

Reply:
For readability. I didn't ask for it to be exported - just that it be declared as a type above and used here. However, this is your call.

Reply:
I think readability is affected by consistency more than pulling things out. As we start using prettier everywhere, this becomes super easy to scan inline, instead of having devs jump around to definitions. Would prefer to stick to this convention (unless there's a need to reuse)